pkg/imgutil/imgutil.go

```diff
@@ -369,7 +370,7 @@ func ParseRepoTag(imgName string) (string, string) {
 
 	ref, err := refdocker.ParseDockerRef(imgName)
 	if err != nil {
-		logrus.WithError(err).Warnf("unparsable image name %q", imgName)
+		logrus.WithError(err).Debugf("unparsable image name %q", imgName)
```

Comment:
?

Reply:
Every time there is a <none> image listed out; this Warning is printed out; I think we should not do it; now that we are adding more support for dangling images.